app/src/main/java/io/homeassistant/companion/android/onboarding/authentication/AuthenticationFragment.kt

```diff
@@ -68,17 +71,31 @@ class AuthenticationFragment : Fragment() {
                             settings.domStorageEnabled = true
                             settings.userAgentString = settings.userAgentString + " ${HomeAssistantApis.USER_AGENT_STRING}"
                             webViewClient = object : TLSWebViewClient(keyChainRepository) {
+                                @Deprecated("Deprecated in Java")
                                 override fun shouldOverrideUrlLoading(view: WebView?, url: String): Boolean {
                                     return onRedirect(url)
                                 }
 
+                                @RequiresApi(Build.VERSION_CODES.M)
                                 override fun onReceivedError(
                                     view: WebView?,
                                     request: WebResourceRequest?,
                                     error: WebResourceError?
                                 ) {
                                     super.onReceivedError(view, request, error)
-                                    showError(commonR.string.webview_error, null, error)
+                                    Log.e(TAG, "onReceivedError: Status Code: ${error?.errorCode} Description: ${error?.description}")
```

Comment:
The `if (request?.url?.toString() == authUrl)` check should probably also be added to here, it will report non-HTTP errors (like timeouts) for all requests here.

Reply:
> The `if (request?.url?.toString() == authUrl)` check should probably also be added to here, it will report non-HTTP errors (like timeouts) for all requests here.

Good call, this would match our current `WebViewActivity` as well. Well not 100% but close enough lol.



> And maybe also for SSL errors? Although that one is unlikely to fail only partially.

We could add it but I don't see the same check done in `WebViewActivity` so not sure if its needed?

Reply:
> We could add it but I don't see the same check done in `WebViewActivity` so not sure if its needed?

Yep it's a bit much for something that is unlikely to happen, if there haven't been any complaints about the behavior in the `WebViewActivity` it should be fine as it is.